usaspending_api/api_contracts/contracts/award/idv/FundingAccounts.md

```diff
@@ -0,0 +1,46 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Funding Treemap
+
+These endpoints are used to power USAspending.gov's IDV Summary Funding Treemap component.
+
+## List Federal Accounts [/api/v2/awards/idvs/accounts/]
+
+This endpoint returns a list of federal accounts under a given IDV.
+
+
+### List Federal Accounts [POST]
++ Request (application/json)
+    + Attributes
+        + `award_id`:`CONT_AW_4732_-NONE-_GS30FHA006_-NONE-` (required, string) 
+            IDV to return accounts for
+        + `page`: 1 (optional, number)
+            Page number to return
+        + `limit`: 10 (optional, number)
+            Maximum number to return
+        + `order`: `desc` (optional, string)
+            Direction of sort
+        + `sort`:`total_transaction_obligated_amount` (optional, string)
+            The field to sort on
+
++ Response 200 (application/json)
+    + Attributes 
+       + page_metadata: (required, PageMetadata)
+       + results: (required, array[AccountListing])
+
+
+# Data Structures
+
+## PageMetadata (object)
++ page: 1 (required, number)
++ next: 2 (required, number, nullable)
++ count: 4 (required, number)
++ previous: 1 (required, number, nullable)
++ hasNext: true (required, boolean)
++ hasPrevious: false (required, boolean)
+
+## AccountListing (object)
++ total_transaction_obligated_amount: 1234.56 (required, number)
++ federal_acount: 075-1301 (required, string)
++ account_title: National Institutes of Health (required, string)
```

Comment:
The [latest mockup](https://bahdigital.invisionapp.com/share/69IA8EPGPCM#/296008376_Award_Summary_2-0_-_IDV_-Future-_-Fed_Acct_List_View-) includes funding agency links as well. Will that be added in a future iteration?

Reply:
@ebdabbs correct, ticket to be created shortly

Reply:
@ebdabbs correct, ticket to be created shortly